src/plugins/list/README.md

```diff
@@ -0,0 +1,63 @@
+- infos = Information about the list plugin is in keys below
+- infos/author = Thomas Waser <thomas.waser@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/placements = presetstorage pregetstorage postgetstorage precommit postcommit prerollback postrollback
+- infos/description =
+
+## Introduction ##
+
+The List plugin is a workaround for the placement slot limitation. It takes a list of plugins + configurations 
```

Comment:
Its not limited to that, it can also be used everywhere a list of plugins is needed.